charts/celeborn/templates/worker/statefulset.yaml

```
@@ -54,8 +54,13 @@ spec:
       {{- end }}
       {{- end }}
       {{- end }}
-      {{- if $paths }}
+      {{- if or .Values.worker.extraInitContainers $paths }}
       initContainers:
+      {{- end }}
+        {{- with .Values.worker.extraInitContainers }}
+        {{- toYaml . | nindent 6 }}
+        {{- end }}
```

LGTM! Just a tiny format nit, I prefer no array indentation in Kubernetes yaml manifests.

```suggestion
      {{- if or .Values.worker.extraInitContainers $paths }}
      initContainers:
      {{- end }}
      {{- with .Values.worker.extraInitContainers }}
      {{- toYaml . | nindent 6 }}
      {{- end }}
```

It would be better to be consistent in indentation with other array fields, for example, `env` and `envFrom`:

https://github.com/apache/celeborn/blob/e5cef163646ff4592d981aad8eb714ccd3833414/charts/celeborn/templates/worker/statefulset.yaml#L106-L113